pep-0670.rst

```diff
@@ -183,13 +186,16 @@ Using static inline functions in the internal C API is fine: the
 internal C API exposes implementation details by design and should not be
 used outside Python.
 
+The performance impact must be measured and regarded before converting a macro
+to a static inline function or a regular function.
```

Comment:
The first paragraph of this section is not enough?

I'm not aware of any performance slowdown of converting a macro to a static inline function. I wouldn't require to run a benchmark for this case.

Reply:
I guess the first paragraph is enough. There is no need in repeating stuff already said.